projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -102,9 +105,128 @@ ${ issueReferences
+	}
+
 	return commentBody;
 }
 
+/**
+ * Build an object containing the slack message and its formatting to send to Slack.
+ *
+ * @param {WebhookPayloadIssue} payload - Issue event payload.
+ * @param {string}              channel - Slack channel ID.
+ * @param {string}              message - Basic message (without the formatting).
+ * @returns {Object} Object containing the slack message and its formatting.
+ */
+function formatSlackMessage( payload, channel, message ) {
+	const { issue, repository } = payload;
+	const { html_url, title } = issue;
+
+	let dris = '@bug_herders';
+	switch ( repository.full_name ) {
+		case 'Automattic/jetpack':
+			dris = '@jpop-da';
+			break;
+		case 'Automattic/zero-bs-crm':
+			dris = '@heysatellite';
+			break;
+	}
+
+	return {
+		channel,
+		blocks: [
+			{
+				type: 'section',
+				text: {
+					type: 'mrkdwn',
+					text: message,
+				},
+			},
+			{
+				type: 'divider',
+			},
+			{
+				type: 'section',
+				text: {
+					type: 'mrkdwn',
+					text: `${ dris } Could you take a look at it, re-prioritize and escalate if you think that's necessary, and mark this message as :done: once you've done so? Thank you!`,
+				},
+			},
+			{
+				type: 'divider',
+			},
+			{
+				type: 'section',
+				text: {
+					type: 'mrkdwn',
+					text: `<${ html_url }|${ title }>`,
+				},
+				accessory: {
+					type: 'button',
+					text: {
+						type: 'plain_text',
+						text: 'View',
+						emoji: true,
+					},
+					value: 'click_review',
+					url: `${ html_url }`,
+					action_id: 'button-action',
+				},
+			},
+		],
+		text: `${ message } -- <${ html_url }|${ title }>`, // Fallback text for display in notifications.
+		mrkdwn: true, // Formatting of the fallback text.
+	};
+}
+
+/**
+ * Check if an issue needs to be escalated to the triage team.
+ *
+ * If our issue has gathered more than 10 tickets,
+ * if we have the necessary Slack tokens,
+ * if we didn't send anything to Slack about that issue yet,
+ * let's send a Slack message to warn the triage team,
+ * and make a note so we don't send it again.
+ *
+ * @param {Array}               issueReferences - Array of support references.
+ * @param {string}              commentBody     - Previous comment ID.
+ * @param {WebhookPayloadIssue} payload         - Issue event payload.
+ * @returns {Promise<boolean>} Was the issue escalated?
+ */
+async function checkForEscalation( issueReferences, commentBody, payload ) {
+	// No Slack tokens, we won't be able to escalate. Bail.
+	const slackToken = getInput( 'slack_token' );
+	const channel = getInput( 'slack_quality_channel' );
+	if ( ! slackToken || ! channel ) {
+		return false;
+	}
+
+	// Issue hasn't gathered more than 10 tickets yet, bail.
+	if ( issueReferences.length < 10 ) {
+		return false;
+	}
+
+	// We already sent a Slack message about this issue, bail.
+	if ( commentBody.includes( 'Note: a notification was sent for this issue.' ) ) {
```

Comment:
Potential conditional miss if we'll update the message in https://github.com/Automattic/jetpack/pull/25388/files#diff-f64aa7317027a073493f1e2f63fa142c5fafbc900fccb8d6a08286be6297ba70R110, but forgot to update it here.

Reply:
That's a good point. 80d1ffcb7ac4230b4304060386f968f2bcf87dbb should take care of that.